libbeat/common/file/rotator_test.go

```diff
@@ -216,15 +216,15 @@ func TestRotateDateSuffix(t *testing.T) {
 	firstExpectedPattern := fmt.Sprintf("%s-%s.*", logname, time.Now().Format("20060102150405"))
 	AssertDirContentsPattern(t, dir, firstExpectedPattern)
 
-	time.Sleep(1500 * time.Millisecond)
+	time.Sleep(2 * time.Second)
```

Comment:
I understand this is a quick fix, but in general this is an antipattern as it elongates the test execution. Do you think it's possible to replace it with mocked `time` instance?

Reply:
It is indeed not the best solution. It is possible to refactor the code, however, I do not think it worth saving a few seconds when the whole build process for touching `libbeat` takes 2-3 hours.

Reply:
I understand your concerns, but it's definitely a code smell. I suggest to open a followup Beats issue to fix this test.